formulas/wick-base/functions/wick-make-user

```diff
@@ -67,26 +70,26 @@ wickMakeUser() {
         wickInfo "Creating user: $username"
         args=(useradd)
 
-        if [[ ! -z "$setHome" ]]; then
+        if [[ -n "$setHome" ]]; then
             args[${#args[@]}]=-d
-            args[${#args[@]}]=$setHome
+            args[${#args[@]}]="$setHome"
```

Comment:
Quoting here is unnecessary because it is never changed into a string and interpreted.  Same comment elsewhere in this file.  I thought shellcheck mentioned that, perhaps I was wrong.